src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -91,7 +101,7 @@ This component has the following slot:
 					:style="styles.avatar"
 					:disable-tooltip="true"
 					:disable-menu="true"
-					v-bind="$props"
+					v-bind="$attrs"
```

Comment:
I haven't tested this specific place, but this looks like a behavior change. Is it migration-related?

For example, prop `showUserStatus` prop is unused now.

Probably, we need to fix this on `main` first

Reply:
Indeed, that's a bit inconsistent. I changed it now to explicitly only forward the props that `NcAvatar` uses. If this is ok, I would send a PR to `master` as well.

Reply:
We should anyway wait until https://github.com/nextcloud-libraries/nextcloud-vue/pull/4743 is merged and backported before continuing here.

Reply:
See #4756 for the fix for `master`.

Reply:
With #4756 merged into `next`, the problem should be solved.